instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -166,4 +171,14 @@ public Properties asJavaProperties() {
   public boolean isAgentDebugEnabled() {
     return getBooleanProperty("otel.javaagent.debug", false);
   }
+
+  public void updateLogCaptureThreshold(String property, String value) {
+    if (LOG_CAPTURE_THRESHOLD.equals(property)) {
+      String level = getProperty(property);
+      if (value != null && !value.isEmpty() && !value.equalsIgnoreCase(level)) {
+        getAllProperties().replace(LOG_CAPTURE_THRESHOLD, level, value.toUpperCase());
+        logger.info("Update {} from {} to {}", LOG_CAPTURE_THRESHOLD, level, value);
+      }
+    }
+  }
```

Comment:
```suggestion
  public void updateLogCaptureThreshold(String value) {
    // TODO add test for dynamically updating in each logging instrumentation module
    getAllProperties().put(LOG_CAPTURE_THRESHOLD, value.toUpperCase());
  }
```

Reply:
i don't think it should get updated when value is empty or null.  That would turn the level OFF. Updated.